### ISSUE TITLE
Launcher not pulling all files for GAMMA. 

### ISSUE THREAD

**Issue by ConvexTlc97** [Mord3rca/gamma-launcher#82]
gamma-launcher check-md5 --gamma '/home/tc/Games/Anomoly/GAMMA RC3/' 
OR
gamma-launcher check-md5 --gamma '/home/tc/Games/Anomoly/GAMMA RC3/' --update-cache 
AND
gamma-launcher full-install --anomaly '/home/tc/Games/Anomoly/Anomaly-1.5.1.2/' --gamma '/home/tc/Games/Anomoly/GAMMA RC3/'

All commands seem to not work at properly downloading all GAMMA mod files for the install process. only get 50 some mods and the GAMMA profile is also not downloaded for MO2.

**Comment by ConvexTlc97** [comment]
Used older and newer version of GAMMA RC3 files since its now the 6.0 installer and issues persist with both. have to use the older RC3 folder I backed up to at least keep all my mods and play an older 0.9.1 version for now without any of the new hot fixs and Christmas updates.

**Comment by ConvexTlc97** [comment]
double checked and updated the launcher to the new 1.5 and still dont get the full current 446 files. stuck at 437. the 6.0 RC3 still only gets about 53 to load in MO2.

**Comment by Mord3rca** [comment]
Hi,

this is a known issue (#68) currently working on it

**Comment by maybeZion** [comment]
Posting this here since #68 was closed but the issue persists. With the latest version (445 mods in MO2) and 6.0 RC3, I used git diff to check my 
[modlist.txt](https://github.com/Mord3rca/gamma-launcher/files/13793651/modlist.txt) as someone mentioned earlier... My install is only missing Petrenka's Improved Russian Translation, BUT there are several unactivated mods in my MO2 that are not present in the official GAMMA modlist; These are January PDA crash fix, Companions Rework, and Armors Balancing. I also have No Valid Game Data (Red X in MO2) next to Hideout Furniture Expansion.
